tests/python/pants_test/tasks/test_changed_target_integration.py

```diff
@@ -1,149 +0,0 @@
-# Copyright 2014 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
This shouldn't be removed because we still have the functionality in v2. Possibly, we will rewrite the test, but we need to audit our coverage of this functionality.

Reply:
Presumably this is testing the v1 implementation though? Or no?

Reply:
No, the `--changed` mechanism was redesigned to be implemented in the engine and then passed down to v1. However, we do have `tests/python/pants_test/integration/changed_integration_test.py` and it's possible that that is comprehensive enough to remove this file.

Reply:
I'll leave it for now